### DISCUSSION TITLE
Problem Task ideas (ongoing bucket list)

### DISCUSSION THREAD

**Discussion by colgreen** [colgreen/sharpneat#60] [General]
* Generative sinewave, with more complex waveform
    * e.g. y =sin(x)+sin(1.2 * x) 

  * Generative images task(s)
    * Maximize jpeg size
    * Simple inputs, e.g. generate red/green/blue circle/square/triange

  * NMIST handwritten digits - image generation.
    * acyclic & cyclic (with different number of cycles per activation)
    * Associative vs generative. (Inputs indicating digits 0-9)
    * Use CPPNs?

  * 2D race track. Box2D?

**Comment by tlemo** [comment]
> 2D race track. Box2D?

Not directly reusable (C++ implementation using a custom box2d fork), but just in case it helps:
https://github.com/tlemo/darwin/blob/master/CHANGELOG.md#dec-21st-2019
https://github.com/tlemo/darwin/tree/master/domains/car_track
